R/group.R

```diff
@@ -113,6 +110,18 @@ dyGroup <- function(dygraph,
                       plotter = plotter)
     return(dygraph)
   }
+  
+  # when setting up the first color, we start handling colors here 
+  if(!is.null(color) && is.null(dygraph$x$attrs$colors)){
```

Comment:
Place a space before left parentheses, except in a function call ([Source](http://adv-r.had.co.nz/Style.html)). It should be
```r
if (!is.null(color) && is.null(dygraph$x$attrs$colors)) {
```